### PR TITLE
Feat/cmd echaindb

### PR DESCRIPTION
**Pull request by meowsbits**
Creates a command `echaindb` which does chaindata database inspection and management (read, write). 

Currently has commands `read-chainconfig` and `write-chainconfig`.

```shell
> ./build/bin/echaindb --chaindb ~/SAMSUNG_T5/e/clients/multigeth/classic/geth/chaindata read-chainconfig 0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3
2020/02/01 01:21:36 Opening database...
{
    "networkId": 1,
    "chainId": 61,
    "eip2FBlock": 1150000,
    "eip7FBlock": 1150000,
    "eip150Block": 2500000,
    "eip155Block": 3000000,
    "eip160Block": 3000000,
    "eip161FBlock": 8772000,
    "eip170FBlock": 8772000,
    "eip100FBlock": 8772000,
    "eip140FBlock": 8772000,
    "eip198FBlock": 8772000,
    "eip211FBlock": 8772000,
    "eip212FBlock": 8772000,
    "eip213FBlock": 8772000,
    "eip214FBlock": 8772000,
    "eip658FBlock": 8772000,
    "eip145FBlock": 9573000,
    "eip1014FBlock": 9573000,
    "eip1052FBlock": 9573000,
    "eip152FBlock": 10500839,
    "eip1108FBlock": 10500839,
    "eip1344FBlock": 10500839,
    "eip2028FBlock": 10500839,
    "eip2200FBlock": 10500839,
    "ecip1010PauseBlock": 3000000,
    "ecip1010Length": 2000000,
    "ecip1017FBlock": 5000000,
    "ecip1017EraRounds": 5000000,
    "disposalBlock": 5900000,
    "ethash": {},
    "trustedCheckpoint": null,
    "trustedCheckpointOracle": null,
    "requireBlockHashes": {
        "1920000": "0x94365e3a8c0b35089c1d1195081fe7489b528a84b22199c916180db8b28ade7f",
        "2500000": "0xca12c63534f565899681965528d536c52cb05b7c48e269c2a6cb77ad864d878a"
    }
}
```

```shell
> ./build/bin/echaindb --chaindb ~/SAMSUNG_T5/e/chaindata-archives/multigeth/classic/geth.bak/chaindata write-chainconfig 0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3 < .ia/myconfig.json 
```

### CODE REVIEW

File: cmd/echaindb/cmd/readChainconfig.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
```
Comment (tzdybal) [inline cmd/echaindb/cmd/readChainconfig.go:2]: TODO

File: cmd/echaindb/LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (tzdybal) [inline cmd/echaindb/LICENSE]: TODO

**Review by tzdybal — COMMENTED** [review]
License is not filled with anything (both master file, and boilerplates). 
I don't see a good reason for `viper` usage, but if it's already there, leave it as is ;)

Other than that, it looks good.

**Comment by tzdybal** [comment]
My configuration looks completely different:
```
{
    "networkId": 1,
    "chainId": 61,
    "daoForkBlock": 1920000,
    "eip150Block": 2500000,
    "eip155Block": 3000000,
    "eip160Block": 3000000,
    "petersburgBlock": 9573000,
    "ecip1010PauseBlock": 3000000,
    "ecip1010Length": 2000000,
    "ecip1017FBlock": 5000000,
    "ecip1017EraRounds": 5000000,
    "disposalBlock": 5900000,
    "ethash": {},
    "trustedCheckpoint": null,
    "trustedCheckpointOracle": null,
    "requireBlockHashes": null
}
```
It shows clearly that my db was created with "old school" way of defining forks.

**Review by tzdybal — COMMENTED** [review]
There are still license boilerplates in particular files, which we should at least remove, or better replace with GPLv3 boilerplate.

File: cmd/echaindb/LICENSE (hunk 2 above)
Comment (meowsbits) [reply cmd/echaindb/LICENSE]: I just `rm`ed the LICENSE. https://github.com/etclabscore/multi-geth/pull/133/commits/2d7416fea26f8099ffd42c0435ece62153d192e5

File: cmd/echaindb/cmd/readChainconfig.go (hunk 1 above)
Comment (meowsbits) [reply cmd/echaindb/cmd/readChainconfig.go:2]: I just `rm`ed the LICENSE. https://github.com/etclabscore/multi-geth/pull/133/commits/2d7416fea26f8099ffd42c0435ece62153d192e5

**Comment by meowsbits** [comment]
Removed LICENSE. Prints raw JSON from db now, instead of putting it through the `Unmarshal`er first.

**Pull request merged**